ts/components/Pinpad/index.tsx

```diff
@@ -97,6 +101,20 @@ class Pinpad extends React.PureComponent<Props, State> {
 
       if (!isValid && this.props.clearOnInvalid) {
         this.clear();
+        // disable click keypad
+        this.setState({
+          isDisabled: true
+        });
+        // enable after 1 second
+        setTimeout(() => {
```

Comment:
this timeout must be cancelled in `componentWillUnmount` to avoid [this](https://www.robinwieruch.de/react-warning-cant-call-setstate-on-an-unmounted-component/)